### PR TITLE
Fix up docblocks and code cleanup.

### PR DESCRIPTION
**Pull request by dereuromark**
What the IDE suggested as cleanup

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -1597,7 +1597,7 @@ public function checkbox(string $fieldName, array $options = []): array|string
      *   the radio label will be 'empty'. Set this option to a string to control the label value.
      *
      * @param string $fieldName Name of a field, like this "modelname.fieldname"
-     * @param iterable $options Radio button options array.
+     * @param iterable|array $options Radio button options array.
```
Comment (ADmad) [inline src/View/Helper/FormHelper.php]: This is incorrect, `iterable` is already `array|Traversable` https://www.php.net/manual/en/language.types.iterable.php

File: src/Console/CommandRunner.php [hunk 2]
```diff
@@ -348,9 +348,7 @@ protected function createCommand(string $className): CommandInterface
             }
 
             $this->factory = new CommandFactory($container);
-            if ($container !== null) {
```
Comment (LordSimal) [inline src/Console/CommandRunner.php:351]: I'd guess we have many more of those in the core. Is there maybe a stan rule out there which would detect those kind of null-safe operator cleanups?

Comment (dereuromark) [reply src/Console/CommandRunner.php:351]: I checked, and @ADmad did most of them, after these they are all done it seems.

Comment (LordSimal) [reply src/Console/CommandRunner.php:351]: I am thinking more of a failsafe net that we don't produce those `!== null` checks in the future.

Comment (dereuromark) [reply src/Console/CommandRunner.php:351]: probably a rector rule, but that PR is still pending to be added to CI.

Comment (dereuromark) [reply src/Console/CommandRunner.php:351]: According to chatgpt also a custom phpstan rule could work?
```php
<?php

declare(strict_types=1);

namespace App\PHPStan;

use PhpParser\Node;
use PhpParser\Node\Expr\PropertyFetch;
use PhpParser\Node\Stmt\If_;
use PHPStan\Analyser\Scope;
use PHPStan\Rules\Rule;

class DetectIfCacheFetchRule implements Rule
{
    public function getNodeType(): string
    {
        return If_::class;
    }

    public function processNode(Node $node, Scope $scope): array
    {
        if (!$node instanceof If_) {
            return [];
        }

        // Check for: if ($this->_cache) { $results = $this->_cache->fetch($this); }
        if ($node->cond instanceof PropertyFetch &&
            $node->stmts[0] instanceof Node\Stmt\Expression &&
            $node->stmts[0]->expr instanceof Node\Expr\MethodCall) {
            
            $propertyFetch = $node->cond; // $this->_cache
            $methodCall = $node->stmts[0]->expr; // $this->_cache->fetch($this)
            
            // Match the conditional and the method call
            if ($this->areNodesEqual($propertyFetch, $methodCall->var)) {
                return ['Consider refactoring to use the null safe operator (?->).'];
            }
        }

        return [];
    }

    private function areNodesEqual(Node $node1, Node $node2): bool
    {
        // Implement node comparison logic
        return $node1->getAttributes() == $node2->getAttributes();
    }
}
```

Comment (dereuromark) [reply src/Console/CommandRunner.php:351]: But in the end this wouldn't be a big deal even if so. Minor refactoring either way.

**Review by ADmad — APPROVED** [review]

**Pull request merged**